scaleway/resource_baremetal_server.go

```diff
@@ -159,6 +159,64 @@ If this behaviour is wanted, please set 'reinstall_on_ssh_key_changes' argument
 					},
 				},
 			},
+			"ipv4": {
+				Type:        schema.TypeList,
+				Computed:    true,
+				Description: "IPv4 addresses attached to the server",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"id": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "The ID of the IPv4",
+						},
+						"version": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "The version of the IPv4",
+						},
+						"address": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "The IPv4 address",
+						},
+						"reverse": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "The Reverse of the IPv4",
+						},
+					},
+				},
+			},
+			"ipv6": {
+				Type:        schema.TypeList,
+				Computed:    true,
+				Description: "IPv6 addresses attached to the server",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"id": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "The ID of the IPv6",
+						},
+						"version": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "The version of the IPv6",
+						},
+						"address": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "The IPv6 address",
+						},
+						"reverse": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "The Reverse of the IPv6",
+						},
+					},
+				},
```

Comment:
Put it in another function since it's duplicated in `ipv4` and `ipv6` (maybe `resourceScalewayBaremetalServerIP`?)